llabake/Hello-Books
# Ft user profile react redux 154423282

#### What does this PR do?
It created the user profile page 
#### Description of Task to be completed?
User should be able to navigate the their profile and view it
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install all the dependency
* Run npm run dev
* visit the port where the webpack-dev-server started
* on successful login, on navigation to the profile page, the user profile page should be loaded there
#### What are the relevant pivotal tracker stories?
154423282
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/35179607-0f444cb2-fd9e-11e7-984c-53be50289bbb.png)
